tfjs-core/src/ops/browser.ts

```diff
@@ -371,4 +386,46 @@ export async function toPixels(
   return bytes;
 }
 
+/**
+ * Draws a `tf.Tensor` to a canvas.
+ *
+ * When the dtype of the input is 'float32', we assume values in the range
+ * [0-1]. Otherwise, when input is 'int32', we assume values in the range
+ * [0-255].
+ *
+ * @param image A rank-2 tensor with shape `[height, width]`, or a rank-3 tensor
+ * of shape `[height, width, numChannels]`. If rank-2, draws grayscale. If
```

Comment:
"draws grayscale" -> "draws in grayscale"
I feel like the comment can be written to be more clear. @mattsoulanille please help us with grammar.

Reply:
Done.